daemon/algod/api/server/v2/generated/types.go

```diff
@@ -600,365 +708,381 @@ type RoundNumber uint64
-	Max *uint64 `json:"max,omitempty"`
+	// Max Max number of box names to return. If max is not set, or max == 0, returns all box-names.
+	Max *uint64 `form:"max,omitempty" json:"max,omitempty"`
 }
 
 // GetBlockParams defines parameters for GetBlock.
 type GetBlockParams struct {
-
-	// Configures whether the response object is JSON or MessagePack encoded.
-	Format *string `json:"format,omitempty"`
+	// Format Configures whether the response object is JSON or MessagePack encoded.
+	Format *GetBlockParamsFormat `form:"format,omitempty" json:"format,omitempty"`
 }
 
+// GetBlockParamsFormat defines parameters for GetBlock.
+type GetBlockParamsFormat string
+
 // GetTransactionProofParams defines parameters for GetTransactionProof.
 type GetTransactionProofParams struct {
-
-	// The type of hash function used to create the proof, must be one of:
+	// Hashtype The type of hash function used to create the proof, must be one of:
 	// * sha512_256
 	// * sha256
-	Hashtype *string `json:"hashtype,omitempty"`
+	Hashtype *GetTransactionProofParamsHashtype `form:"hashtype,omitempty" json:"hashtype,omitempty"`
 
-	// Configures whether the response object is JSON or MessagePack encoded.
-	Format *string `json:"format,omitempty"`
+	// Format Configures whether the response object is JSON or MessagePack encoded.
+	Format *GetTransactionProofParamsFormat `form:"format,omitempty" json:"format,omitempty"`
 }
 
+// GetTransactionProofParamsHashtype defines parameters for GetTransactionProof.
+type GetTransactionProofParamsHashtype string
+
+// GetTransactionProofParamsFormat defines parameters for GetTransactionProof.
+type GetTransactionProofParamsFormat string
+
+// TealCompileTextBody defines parameters for TealCompile.
+type TealCompileTextBody = openapi_types.File
+
 // TealCompileParams defines parameters for TealCompile.
 type TealCompileParams struct {
-
-	// When set to `true`, returns the source map of the program as a JSON. Defaults to `false`.
-	Sourcemap *bool `json:"sourcemap,omitempty"`
+	// Sourcemap When set to `true`, returns the source map of the program as a JSON. Defaults to `false`.
+	Sourcemap *bool `form:"sourcemap,omitempty" json:"sourcemap,omitempty"`
 }
 
-// TealDryrunJSONBody defines parameters for TealDryrun.
-type TealDryrunJSONBody DryrunRequest
-
 // GetPendingTransactionsParams defines parameters for GetPendingTransactions.
 type GetPendingTransactionsParams struct {
+	// Max Truncated number of transactions to display. If max=0, returns all pending txns.
+	Max *uint64 `form:"max,omitempty" json:"max,omitempty"`
 
-	// Truncated number of transactions to display. If max=0, returns all pending txns.
-	Max *uint64 `json:"max,omitempty"`
-
-	// Configures whether the response object is JSON or MessagePack encoded.
-	Format *string `json:"format,omitempty"`
+	// Format Configures whether the response object is JSON or MessagePack encoded.
+	Format *GetPendingTransactionsParamsFormat `form:"format,omitempty" json:"format,omitempty"`
```

Comment:
@Eric-Warehime For my edification:  What uses the `form` tag?

I'm mostly unfamiliar with how to track down usages of struct tags.  I went as far as confirming `form` is not on https://github.com/golang/go/wiki/Well-known-struct-tags.

Reply:
It looks like they added it [in this commit](https://github.com/deepmap/oapi-codegen/commit/6a3482493b5731b60c267147522ec3aaa09fc3a3) and it's discussed in [this issue](https://github.com/deepmap/oapi-codegen/issues/500) a bit. 

The `form` tag is being added here only to data types that are query parameters. Those objects are marshaled using the query parameters provided in requests. In our case it looks like [only the json tag is used](https://github.com/algorand/oapi-codegen/blob/master/pkg/runtime/bindparam.go#L479) so this should be a non-functional change for us.

Reply:
@Eric-Warehime Got it - Resolving.  Appreciate the detailed context!